DesignSafe-CI/portal
# DES-2417: User Data Modal Additions

## Overview: ##
Adding functionality to enable a modal pop-up window with basic user information in the main Projects View and Curation View in the same fashion as in publications.

## PR Status: ##


## Related Jira tickets: ##

* [DES-2417](https://jira.tacc.utexas.edu/browse/DES-2417)

## Summary of Changes: ##
Updated controller code to call UserService to get user info
Added function to controller code for pop-up modal call
Modified prj-metadata template to work for either type of display (there are views where we don't need the pop-up so they can keep using the older template code)

## Testing Steps: ##
1. Open a project and make sure that the user data loads for PI and Co-PI and that when you click a name the modal appears on screen.
2. Check both the Projects View and Curation View.

## UI Photos:

## Notes: ##
Publication Preview does have some of this functionality now, however there's a bug preventing this from working correctly for type Other projects.  The type Other issue will be resolved at a later time.
